javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,73 @@
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1986;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+
+console.log(`You will be ${age} years old in ${yearFuture}`); 
+    
+
+  
+
+// Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth = 2016;
+let dogYearFuture = 2027;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = true;
+    
+if (shouldShowResultInDogYears){
+futureAgeInDogYears = dogYear * 7;
```

Comment:
Remember to declare your variables using let or const
Other than that, Good job 

```suggestion
const futureAgeInDogYears = dogYear * 7;
```

Reply:
Thank you Ricardo I will update it